### PR TITLE
Archive.org Drop jQuery and legacy

### PR DESCRIPTION
**Pull request by inverse**
**Describe the changes you made**

- Drop legacy design (it's been a long while and not supported from what I could see)
- Drop jQuery based selectors

**Pages to test**

- https://archive.org/details/AH013_sarin_sunday_-_the_lonely_hike/01_Light_Fragments.mp3

### CODE REVIEW

File: src/connectors/archive.js [hunk 1]
```diff
@@ -1,89 +1,28 @@
...
 
-	Connector.durationSelector = '.jw-text-duration';
+Connector.trackArtSelector = '#theatre-ia center > img';
 
-	Connector.isPlaying = () => {
-		return !$('video')[0].paused;
-	};
-}
+Connector.getArtistTrack = () => {
+	let track = document.querySelector('.jwrowV2.playing .ttl, .audio-track-list .selected .track-title').textContent;
```
Comment (alexesprit) [inline src/connectors/archive.js:14]: It's better to use array of selectors, as they are much safer.

```js
const artistSelectors = [
	'.key-val-big a span',
	'.metadata-definition > dd',
];
const trackSelectors = [
	'.jwrowV2.playing .ttl',
	'.audio-track-list .selected .track-title',
];

Connector.getArtistTrack = () => {
	let track = Util.getTextFromSelectors(trackSelectors);
	const artist = Util.getTextFromSelectors(artistSelectors);
	...
};
```

File: src/connectors/archive.js [hunk 2]
```diff
@@ -1,89 +1,28 @@
...
-	case 'legacy':
-		bindLegacy();
-		break;
+Connector.durationSelector = '.jw-text-duration';
 
-	default:
-		Util.debugLog('Unable to match version. Please report at https://github.com/web-scrobbler/web-scrobbler/issues', 'warn');
-		break;
-}
+Connector.isPlaying = () => !document.querySelector('video').paused;
```
Comment (alexesprit) [inline src/connectors/archive.js:7]: `document.querySelector` could return `null`, so I would add an additional check here.

**Review by alexesprit — CHANGES_REQUESTED** [review]

**Comment by alexesprit** [comment]
Thanks! I've already rewritten all of them, but still happy to merge this PR.

File: src/connectors/archive.js (hunk 1 above)
Comment (inverse) [reply src/connectors/archive.js]: I think we should document these Util functions more if we want people to adopt them.

**Comment by inverse** [comment]
@alexesprit no way - on a branch? Isn't that a pain to maintain a long running branch with churn on master?

**Review by alexesprit — APPROVED** [review]

**Pull request merged**